packages/compass-aggregations/src/modules/pipeline-builder/index.ts

```diff
@@ -2,11 +2,13 @@ import { combineReducers } from 'redux';
 import stageEditor from './stage-editor';
 import textEditor from './text-editor';
 import pipelineMode from './pipeline-mode';
+import outputStage from './text-editor-output-stage';
 
 const reducer = combineReducers({
   pipelineMode,
   stageEditor,
   textEditor,
+  outputStage,
```

Comment:
Feel free to keep as-is, but from my perspective putting it outside of text editor even when text editor is the only user of this slice makes it a bit confusing: makes it look like something else might rely on this state, like stage editor, where in reality it is not

Reply:
I started that way, but then separated the two. I did not want to mix preview pipeline and actually running of pipeline. 

With another point you raised regarding `serverError` prop,  I kinda feel like i should put this in `textEditor` slice.

Reply:
Hmmm, if you want to keep them split, maybe would make sense to restructure stuff a little and make text-editor a combined reducer itself? Something like `{ textEditor: { pipeline: <pipeline editor state>, output: <preview and out / merge state> } }` maybe?